app/controllers/menus_controller.rb

```diff
@@ -2,6 +2,11 @@ class MenusController < ApplicationController
 
   def show
     @restaurant = Restaurant.find(params[:restaurant_id])
-    @cart_nr = ShoppingCartItem.count
+    if session[:cart_id]
+      @cart = ShoppingCart.find(session[:cart_id])
+      @cart_nr = @cart.shopping_cart_items.count
```

Comment:
We are going to move these later and it is out of scope as well but I see what you mean and will modify it when doing the nav bar.